tests/linearGaussian_snpe_test.py

```diff
@@ -41,12 +41,12 @@
 )
 
 
-@pytest.mark.parametrize("npe_method", [NPE_A, NPE_C])
+@pytest.mark.parametrize("npe_method_str", ["npe_a", "npe_b", "npe_c"])
```

Comment:
I think you changed this to a string so that you can pass it to `check_c2st`, but I think it's actually better to parameterize the methods directly. A quick fix to make sure that the correct string gets used for the `check_c2st` print statement is to add a `__name__` attribute to the `trainer` methods, and then use `npe_method.__name__` here.